exchange_machine.lua

```diff
@@ -225,32 +215,28 @@ minetest.after(0, function()
 		inv:add_item("main", ItemStack(itemstring))
 	end
 
-	selectable_inventory_size = #selectable_list
+	pagemax = math.ceil((#selectable_list - 1) / (8 * 4))
 end)
 
 
 minetest.register_on_joinplayer(function(player)
-	local big_inv = minetest.get_inventory({type="detached", name="global_exchange"})
-	local p_list = "p_" .. player:get_player_name()
```

Comment:
The reason I assigned this separate originally is to document what that string is supposed to be